### DISCUSSION TITLE
Example Repository for Implementing OpenCV with YOLO

### DISCUSSION THREAD

**Discussion by pdcr42** [community/community#131161] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone !!!

I hope you’re all doing well. I am currently working on a project that involves implementing OpenCV with YOLO (You Only Look Once) for object detection. I have some experience with both OpenCV and YOLO individually, but I am looking for a good example repository that integrates these two technologies effectively.

Could anyone please recommend a GitHub repository or share some sample code that demonstrates the implementation of OpenCV with YOLO?

**Comment by chiarorosa** [comment]
You can find a detailed example of object detection using SSD MobileNetV2 in this excellent article: [Object Detection Using SSD MobileNetV2 Using TensorFlow API](https://medium.com/@techmayank2000/object-detection-using-ssd-mobilenetv2-using-tensorflow-api-can-detect-any-single-class-from-31a31bbd0691). While it focuses on SSD MobileNetV2, the concepts and steps outlined can be quite helpful for understanding the integration process, which you can adapt for YOLO and OpenCV.

**Moreover, there is a GitHub repository that already includes examples related to computer vision tasks, which could be very useful: [GitHub - chiarorosa/visao-computacional](https://github.com/chiarorosa/visao-computacional). This repository provides practical examples and code that demonstrate how to implement various computer vision techniques.**
